### PR TITLE
Adding logo 

### LINKED ISSUE #84: UI: Add dune logo/favicon
https://atwork.dunescience.org/logo-templates/

Maybe we can lift the D in the logo for the favicon?

See #85 for reference.

### PR DESCRIPTION
**Pull request by Sahil590**
# Description

_Please include a summary of the change and which issue is fixed (if any). Please also
include relevant motivation and context. List any dependencies that are required for
this change._

Fixes #84  (issue)

## Type of change

- [ ] Documentation (non-breaking change that adds or improves the documentation)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (non-breaking, back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (whatever its nature)

## Key checklist

- [x] All tests pass (eg. `python -m pytest`)
- [ ] The documentation builds and looks OK (eg. `python -m sphinx -b html docs docs/build`)
- [x] Pre-commit hooks run successfully (eg. `pre-commit run --all-files`)

## Further checks

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added or an issue has been opened to tackle that in the future. (Indicate issue here: # (issue))

### CODE REVIEW

**Review by dalonsoa — APPROVED** [review]
Nice!

File: main/templates/main/base.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {% load static %}
 {% load django_bootstrap5 %}
+<link rel="shortcut icon"
+      type="image/png"
+      href="{% static 'main/images/Favicon.png' %}">
```
Comment (jamesturner246) [inline main/templates/main/base.html:5]: Looks like this typically goes at the bottom of `<head>`, just before `</head>`. See e.g. [here](https://stackoverflow.com/questions/48354130/what-is-the-proper-place-of-favicon-references-in-head).

**Review by jamesturner246 — CHANGES_REQUESTED** [review]
Looks good. :+1:  To be tidy, though, I think favicons go in head .

**Review by jamesturner246 — APPROVED** [review]
LGTM! :+1:

File: main/templates/main/base.html [hunk 2]
```diff
@@ -19,6 +18,10 @@
     {% endblock extra_js %}
     <script src="{% static 'main/js/htmx.min.js' %}"></script>
     <script src="{% static 'main/js/_hyperscript.min.js' %}"></script>
+    <link rel="shortcut icon"
+          type="image/png"
+          href="{% static 'main/images/Favicon.png' %}">
+    <!DOCTYPE html>
```
Comment (cc-a) [inline main/templates/main/base.html:24]: Looks like this has been moved by accident.

Comment (Sahil590) [reply main/templates/main/base.html:24]: If we want to add the logo to the navbar as well, would we keep the "Run Control" name in place something like this 
image
or just remove it all together.

**Review by cc-a — CHANGES_REQUESTED** [review]
I'm getting super picky here but the vertical alignment of "Run Control" is hurting my brain. Can we tweak it to be vertically centered?

Less important but it would also be nice for the horizontal space between the logo, "Run Control" and the buttons to be the same.

**Comment by dalonsoa** [comment]
May I suggest we have a general pass all together to the look and feel and layout, so we are not touching small bits here and there?

File: main/templates/main/navbar.html [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
 <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
   <div class="container">
-    <a class="navbar-brand" href="{% url 'main:index' %}">DUNE Run Control</a>
+    <a class="navbar-brand" href="{% url 'main:index' %}">
+      <img src="{% static 'main/images/DUNElogo_white.png' %}"
+           alt="DUNE Run Control"
+           width="80"
+           height="50"
+           class="d-inline-block align-top">
+      <h2 class="d-inline-block">Run Control</h2>
```
Comment (AdrianDAlessandro) [inline main/templates/main/navbar.html:11]: Maybe following this example more closely will align the text better?

https://getbootstrap.com/docs/5.0/components/navbar/#image-and-text

I guess that's maybe something like this (could also involve changing width and height):
```suggestion
           class="d-inline-block align-text-top">
      Run Control
```

**Comment by Sahil590** [comment]
@cc-a 
image
does this work?

**Comment by cc-a** [comment]
@Sahil590 if you vertically center "Run Control" I think that would be best.

**Review by cc-a — APPROVED** [review]
Grand. Thanks for looking at it again.

**Pull request merged**